src/script/components/MessagesList/Message/ContentMessage/asset/TextMessageRenderer.tsx

```diff
@@ -74,92 +70,69 @@ export const TextMessageRenderer: FC<TextMessageRendererProps> = ({
   }, [isQuoteMsg, setCanShowMore, containerRef]);
 
   useEffect(() => {
-    const element = containerRef;
-    if (!element) {
-      return undefined;
+    if (!containerRef) {
+      return;
```

Comment:
Previously, you were returning an arrow function to get rid of the event listeners. But now that this is gone, this function returns nothing and typescript is happy because this function is never returning anything :)